### PR TITLE
Added CloudFormation, deleted Cloudwatch from table

### PR DESCRIPTION
**Pull request by chrisnegus**
*Description of changes:*
Added CloudFormation and AWS CDK to EKS-A/EKS comparison table. Removed CloudWatch.

### CODE REVIEW

**Comment by chrisnegus** [comment]
/hold
for approval from @vipin-mohan

**Comment by chrisnegus** [comment]
/unhold
This is ready to merge, when someone is available to okay it.

**Review by abhay-krishna — APPROVED** [review]
/lgtm

**Pull request merged**
